### ISSUE TITLE
[Question] Is it possible to disable the rule `format-comment` for some patterns (ex regexp) ?

### ISSUE THREAD

**Issue by ValentinVignal** [dart-code-checker/dart-code-metrics#826]
I'm using the rule `format-comment`.

I'm working on VSCode with the extension cSpell: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker that checks the spelling in the code.

Sometime I have to write:

```dart
// cSpell:ignore myunknownword
```

or 

```dart
// cSpell:disable-next-line
```

And I get a lint on it.

Is it possible to disable the rule for some patterns?

**Comment by incendial** [comment]
Nope, it's not possible, feel free to open a PR with this improvement. If you have any questions, I'd be happy to help.

**Comment by ValentinVignal** [comment]
Thank you! I can try to make a PR. I guess I would have to modify the file `lib/src/analyzers/lint_analyzer/rules/rules_list/format_comment/visitor.dart`.

I was thinking to add an option:

```yaml
dart_code_metrics:
  rules:
    format-comment:
      ignored-patterns:
        - myRegExp1
        - myRegExp2
```

What do you think?


But I'm a bit lost on how to add this option and access it from `lib/src/analyzers/lint_analyzer/rules/rules_list/format_comment/visitor.dart`.

**Comment by ValentinVignal** [comment]
I tried to do something here: https://github.com/dart-code-checker/dart-code-metrics/pull/828

Tell me if I
- miss anything
- did something wrong
- should do something differently

**Comment by incendial** [comment]
@ValentinVignal great job! 🔥 Left few small comments in the PR

**Comment by incendial** [comment]
Available in 4.15.2 🚀
